src/plugins/config/Config.tsx

```diff
@@ -24,6 +24,11 @@ const toolbar = (theme: Theme) => css`
     ${theme.typography.pxToRem(theme.spacing(4))};
 `;
 
+const varDescription = (theme: Theme) => css`
+  padding: ${theme.typography.pxToRem(theme.spacing(2))}
```

Comment:
I think i would actually make this margin. 

and add `margin-bottom: 0`; 

```js
const varDescription = (theme: Theme) => css`
  margin: ${theme.typography.pxToRem(theme.spacing(2))}
    ${theme.typography.pxToRem(theme.spacing(4))};
  margin-bottom: 0;
```

Reply:
Looks great now